examples/function_call/role_playing_with_functions.py

```diff
@@ -60,15 +60,15 @@ def main(model_type=ModelType.GPT_3_5_TURBO, chat_turn_limit=10) -> None:
         assistant_agent_kwargs=dict(
             model=ModelFactory.create(
                 model_platform=ModelPlatformType.OPENAI,
-                model_type=ModelType.GPT_3_5_TURBO,
+                model_type=model_type,
                 model_config_dict=assistant_model_config.__dict__,
             ),
             tools=function_list,
         ),
         user_agent_kwargs=dict(
             model=ModelFactory.create(
                 model_platform=ModelPlatformType.OPENAI,
-                model_type=ModelType.GPT_3_5_TURBO,
+                model_type=model_type,
                 model_config_dict=user_model_config.__dict__,
             ),
         ),
```

Comment:
There will be a minor bug if the input parameter `model_type` is not an OpenAI-based model. We need to ensure that `model_platform` and `model_type` match.